drivers/hmis_csv_importer/app/models/hmis_csv_importer/import_override.rb

```diff
@@ -0,0 +1,86 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class HmisCsvImporter::ImportOverride < GrdaWarehouseBase
+  belongs_to :data_source
+  has_paper_trail
+  acts_as_paranoid
+  belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
+
+  scope :sorted, -> do
+    order(:file_name, :matched_hud_key)
+  end
+
+  def self.available_classes
+    Importers::HmisAutoMigrate.available_migrations.values.last.constantize::TRANSFORM_TYPES
+  end
+
+  def self.available_files
+    available_classes.keys
+  end
+
+  def self.known_columns
+    [
+      :file_name,
+      :matched_hud_key,
+      :replaces_column,
+      :replaces_value,
+      :replacement_value,
+    ]
+  end
+
+  # Accepts either an object based on GrdaWarehouse::Hud::Base, or a has of attributes with string keys
+  # Returns same object with overides applied
+  # NOTE: this does not save the object
+  def apply(row)
+    return row unless applies?(row)
+
+    # We either have the right HUD Key, or the right source value, or both
+    # or we weren't looking for anything specific
+    # Just replace the data
+    row[replaces_column] = replacement_value
+
+    row
+  end
+
+  private def normalize_row(row)
+    return row.attributes if row.is_a?(GrdaWarehouse::Hud::Base)
+
+    row
+  end
+
+  def applies?(row)
+    row = normalize_row(row)
+    # Double check we actually have the column we're looking for (protects against typos or future spec changes)
+    return false unless row.key?(replaces_column)
+    # We were expecting a specific HUD key, and this is not it
+    return false if matched_hud_key.present? && row[hud_key] != matched_hud_key
```

Comment:
safe navigation maybe more concise?
```suggestion
    return false unless matched_hud_key.presence&.== row[hud_key]
```

Reply:
Oh, that's not a syntax I'm familiar with.  Not sure what I think about `&.== `.  We'll give it a try!  Though `&.!=` since that's the comparison we're actually making.  Thanks!